canvas.md

```diff
@@ -1,17 +1,58 @@
+8. H. Wang, Y. Shi, "Macroeconomic Determinants of Stock Market Performance: Evidence from Emerging Markets", *Journal of International Financial Markets*, Vol. 68, 2020.
+9. Z. Bai, X. Liu, "Regime-Switching Models in Macro-Finance: Applications to Stock Returns", *Financial Economics Review*, Vol. 24, No. 4, 2017.
+10. Y. Guo, M. Xu, "Machine Learning Approaches in Forecasting Macroeconomic Variables and Stock Returns", *Quantitative Finance*, Vol. 21, No. 1, 2021.
+11. L. Sun, Y. Liu, "Policy Uncertainty and Stock Market Returns: A Global Perspective", *Global Finance Journal*, Vol. 45, 2020.
+12. H. Kim, D. Choi, "Financial Risk Modeling with Macroeconomic Indicators: A Survey", *Finance Research Letters*, Vol. 39, 2021.
+13. X. Zhang, J. Yang, "Economic Policy Uncertainty and Stock Market Volatility: The Role of Investor Sentiment", *Journal of Economic Behavior & Organization*, Vol. 188, 2022.
+14. P. Liu, Z. Zhao, "Cross-Asset Portfolio Optimization Using Macroeconomic Data", *Journal of Portfolio Management*, Vol. 49, No. 3, 2023.
+15. N. Werge, "Predicting Risk-Adjusted Returns Using an Asset Independent Regime-Switching Model", *Expert Systems with Applications*, Vol. 184, 2021.
+16. T. Zheng, H. Ge, "Time-Varying Characteristics of Herding Effects in the Chinese Stock Market: A Regime-Switching Approach", *Journal of Financial Research*, Vol. 44, No. 3, 2021.
+17. P. Andreini, C. Izzo, G. Ricco, "Deep Dynamic Factor Models for Macroeconomic Forecasting and Nowcasting", *Journal of Econometrics*, Vol. 220, No. 2, 2021.
 
 ## Relevance/Significance
 
-* Sequential computation limts parallel processing 
-* ... more points
+* Understanding Market Dynamics
+* Practical Applications
+* Backtest on stock market
 
 ## Originality/Novelty
 
-## Hypothesis/Research Question
\ No newline at end of file
+* Introduces a multi-dimensional quantitative framework combining traditional macroeconomic indicators with market behavior analysis.
+* Employs advanced machine learning methods (e.g., random forests, LSTM) to integrate high-frequency market data and low-frequency macroeconomic data.
+* Develops actionable investment strategies by bridging the gap between theoretical research and real-world applications.
+
+
+## Hypothesis/Research Question
+* Uncertainty and Volatility
+* Which macroeconomic indicators (e.g., GDP, inflation, interest rates) have the most significant impact on asset price volatility?
+* Do markets exhibit structural shifts in volatility following major policy changes or external shocks?
+
+## Results
+
+
+
+## Contributions
+* A comprehensive framework combining macroeconomic indicators and market volatility analysis using machine learning.
+* Quantitative evaluation of the impact of historical uncertainty events on market behavior.
+* Development of a volatility prediction model with direct applications in investment and policy-making.
+
+## Key Collaborators
+* Quantitative Investment Firms

```

Comment:
Be more specific in this section. Do you know of firms? Who are your academic partners? Who are your data providers.